akka-docs/rst/scala/persistence-query.rst

```diff
@@ -0,0 +1,212 @@
+.. _persistence-query-scala:
+
+#################
+Persistence Query
+#################
+
+Akka persistence query complements :ref:`persistence-scala` by providing an universal asynchronous stream based
+query interface that various journal plugins can implement in order to expose their query capabilities.
+
+The most typical use case of persistence query is implementing the so-called query side (also known as "read side"),
+in the popular CQRS architecture pattern - in which the writing side of the application (e.g. implemented using akka
+persistence) is completely separated from the query side which can be implemented using akka persistence query.
```

Comment:
I’d replace “using” by “with the help of” to create more distance—Akka Persistence Query is _not_ an implementation of the read side.